extensions/amp-image-viewer/0.1/amp-image-viewer.js

```diff
@@ -172,13 +172,12 @@ export class AmpImageViewer extends AMP.BaseElement {
       return Promise.resolve();
     }
     const ampImg = dev().assertElement(this.sourceAmpImage_);
-    const isLaidOut = ampImg.hasAttribute('i-amphtml-layout') ||
-      ampImg.classList.contains('i-amphtml-layout');
-    const laidOutPromise = isLaidOut
+    const hasCompletedLayout = !!ampImg.querySelector('img');
```

Comment:
> the first element that is a **descendant** of the element on which it is invoked that matches the specified group of selectors.

Is there any chance that this could have an `img` but still not have completed layout? For example, could this return the fallback or placeholder?

Reply:
Removed the `querySelector` per the other comment.